### PR TITLE
Label/fix

### PR DESCRIPTION
**Pull request by muskan2622**
### Functionality:
Delete label of add member fixing

### Solution:
 UI Fixes

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
Delete Popup label has been fixed ans delete scheduled functionality

### CODE REVIEW

**Review by simran142002 — COMMENTED** [review]
resolve the conflicts

**Review by viv9k — APPROVED** [review]
Approved

**Comment by viv9k** [comment]
Resolve merge conflict

File: functions/model/meet/lib.js [hunk 1]
```diff
@@ -147,3 +161,20 @@ exports.updateMeetDetailsAtWorktez= function(updateJson, meetDocId) {
...
+ * @param {any} docId
+ * @return {any}
+ */
+ exports.getUserMeetDetailsById = function(uid, docId) {
+  const getUserMeetDetailsById = db.collection("Users").doc(uid).collection("Meet").doc(docId).get().then((doc) => {
+    const data = doc.data();
+    return data;
+  });
+  return Promise.resolve(getUserMeetDetailsById);
+};
```
Comment (sanjaykrishna1203) [inline functions/model/meet/lib.js:180]: What is the use of this API when the same data can be fetched using 
`exports.getWorktezMeetDetails` API?

File: functions/model/meet/tark/deleteMeet.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable no-const-assign */
+/* eslint-disable require-jsdoc */
+/* eslint-disable  object-curly-spacing*/
+// /* eslint-disable no-undef */
+/* eslint-disable eol-last */
+/* eslint-disable indent */
+/* eslint-disable max-len */
+// eslint-disable-next-line no-dupe-else-if
```
Comment (sanjaykrishna1203) [inline functions/model/meet/tark/deleteMeet.js:10]: Remove Unnecessary disables 
Not allowed to add additional disables

File: src/app/header/show-meet-details/show-meet-details.component.ts [hunk 3]
```diff
@@ -32,19 +36,26 @@ export class ShowMeetDetailsComponent implements OnInit {
...
+
+
   getMeetData() {
     const uid = this.authService.getLoggedInUser();
     const callable = this.functions.httpsCallable("meet/getMeetDetails");
     callable({ Uid: uid }).pipe(map(actions => {
         return actions.data as Meet[];
       })).subscribe({
         next: (data) => {
+          console.log(data)
```
Comment (sanjaykrishna1203) [inline src/app/header/show-meet-details/show-meet-details.component.ts:58]: Remove console log

File: src/app/header/show-meet-details/show-meet-details.component.ts [hunk 4]
```diff
@@ -57,4 +68,34 @@ export class ShowMeetDetailsComponent implements OnInit {
         }
       })
   }
+
+  setMeetToDelete(item){
+    this.MeetToDelete=item;
+    this.deletedMeetEnabled=true;
+
```
Comment (sanjaykrishna1203) [inline src/app/header/show-meet-details/show-meet-details.component.ts:75]: What is the use of this function?

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]
Please do the required changes

File: functions/model/meet/lib.js (hunk 1 above)
Comment (muskan2622) [reply functions/model/meet/lib.js:180]: Here we are getting users meet.

File: src/app/header/show-meet-details/show-meet-details.component.ts (hunk 4 above)
Comment (muskan2622) [reply src/app/header/show-meet-details/show-meet-details.component.ts:75]: This function is for further scheduling techniques in schedule meet

File: functions/model/meet/lib.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable valid-jsdoc */
```
Comment (sanjaykrishna1203) [inline functions/model/meet/lib.js:2]: Strictly not allowed to add Disables except the allowed four.

File: functions/model/meet/tark/deleteMeet.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    } else {
+      const updateMeetDetailsToJson = {
+        Status: "DELETED",
+      };
+      updateUserMeetDetails(uid, docId, updateMeetDetailsToJson);
+      console.log("meet details check");
+    }
+  }).catch((error) => {
+    // eslint-disable-next-line no-const-assign
+    status = 500;
```
Comment (sanjaykrishna1203) [inline functions/model/meet/tark/deleteMeet.js:41]: Same here.

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: functions/model/meet/tark/deleteMeet.js (hunk 6 above)
Comment (muskan2622) [reply functions/model/meet/tark/deleteMeet.js]: done

Comment (sanjaykrishna1203) [inline functions/model/meet/tark/deleteMeet.js]: Not removed yet..

Comment (muskan2622) [reply functions/model/meet/tark/deleteMeet.js]: I have removed them . Please check

**Review by sanjaykrishna1203 — APPROVED** [review]
Approved

**Pull request merged**